app/logic/validation.py

```diff
@@ -35,6 +35,7 @@ def doesUserHaveRole(role):
   username = getUsernameFromEnv()
   try:
     user = UserQueries.select_single(username)
+    print ("Query working", user)
```

Comment:
Can we remove this print if the function works? 
I think it will get bothersome to have so many prints in the console.